staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go

```diff
@@ -3913,6 +3915,61 @@ func describeStorageClass(sc *storagev1.StorageClass, events *corev1.EventList)
 	})
 }
 
+type CSINodeDescriber struct {
+	clientset.Interface
+}
+
+func (c *CSINodeDescriber) Describe(namespace, name string, describerSettings describe.DescriberSettings) (string, error) {
+	csi, err := c.StorageV1().CSINodes().Get(name, metav1.GetOptions{})
+	if err != nil {
+		return "", err
+	}
+
+	var events *corev1.EventList
+	if describerSettings.ShowEvents {
+		events, _ = c.CoreV1().Events(namespace).Search(scheme.Scheme, csi)
+	}
+
+	return describeCSINode(csi, events)
+}
+
+func describeCSINode(csi *storagev1.CSINode, events *corev1.EventList) (output string, err error) {
+	metadata := csi.GetObjectMeta()
+	return tabbedString(func(out io.Writer) error {
+		w := NewPrefixWriter(out)
+		w.Write(LEVEL_0, "Name:\t%s\n", csi.GetName())
+		printLabelsMultiline(w, "Labels", csi.GetLabels())
+		printAnnotationsMultiline(w, "Annotations", csi.GetAnnotations())
+		w.Write(LEVEL_0, "Metadata:\n")
```

Comment:
Other objects don't have Metadata section, they log `CreationTimestamp` just under `Namespace:` (and it's `CreationTimestamp: `, not `Creation Timestamp:`.

Reply:
I've removed the Metadata section, and adjusted this to be a single word.